docs/setup-cluster/k8s/custom-pod-specs.rst

```diff
@@ -266,3 +266,37 @@ Example ``expconf.yaml``
                  mountPath: /etc/secret-volume
    resources:
      resource_pool: prod_pool
+
+*******************************
+ Custom CheckpointGC Pod Specs
+*******************************
+
+Determined also provides a way to configure CheckpointGC pod specs. This configuration is done using
+the ``task_container_defaults.checkpoint_gc_pod_spec`` field within your ``value.yaml`` file. When a
+custom CheckpointGC pod spec is defined, it takes precedence over the default CheckpointGC pod
```

Comment:
same comment with the release note about clarifying what "default checkpointGC pod" refers to